utils/build-script-impl

```diff
@@ -740,20 +743,26 @@ CROSS_TOOLS_DEPLOYMENT_TARGETS=()
 
 # Determine the native deployment target for the build machine, that will be
 # used to jumpstart the standard library build when cross-compiling.
-case "$(uname -s)" in
-    Linux)
+# FIXME: This will not work while trying to cross-compile
+# It gets the architecture from the system processor arch.
```

Comment:
Can you elaborate on what you mean by this?
The way I understand it, the processor architecture of the build machine is exactly what we'd want here.
Or what exactly is the issue while cross compiling?